static/css/quiz-css.css

```diff
@@ -0,0 +1,53 @@
+.flex-container {
+    display: flex;
+    flex-direction: column;
+}
+
+.title-block {
+    background-color: #4299e1;
+}
+
+.start-quiz-title {
+    text-align: center;
+    color: white;
+    font-weight: bold;
+}
+
+.start-quiz-subtitle {
+    text-align: center;
+    color: white;
+}
+
+.button-bar {
+    display: flex;
+    background-color: #c6f6d5 !important;
+    justify-content: space-between !important;
+
+}
+
+.center-picture {
+    background-color: #bee3f8 !important;
+    display: flex !important;
+    text-align: center;
+    justify-content: center !important;
+}
+
+img {
+   max-width: 25%;
```

Comment:
This CSS rule should be set to a specific class, instead of all images, because it affects all of the app (for example, the Hedy logo).

Reply:
Thanks for the check @fpereiro!

https://github.com/LotusterHaar Let me know when those 2 fixes are ready so I can merge!

Reply:
I fixed both suggestions. But testing it I found a bug: when visiting a site from the middle of the quiz ( so if you haven't started it from the beginning) the session variable session['total_score'] gives a NoneType when it's addeds with question['question_score']. I only have time in the evening to fix that. So I hope it won't be a big problem

Reply:
Hi Lotus! 

We can live with the small issue of leaving the quiz in the middle, so I will test now and merge if all works ok. I will also make an issue for this so we don't forget and assign it to you.